client/state/happychat/constants.js

```diff
@@ -28,3 +28,14 @@ export const HAPPYCHAT_MESSAGE_TYPES = {
 	CUSTOMER_INFO: 'customer-info',
 	LOG: 'log',
 };
+
+// Chat status
```

Comment:
Thank you, this refactor to move these into a `constants` file is LOOONG overdue :)